$OpenBSD: patch-src_librustc_binaryen_build_rs,v 1.1 2018/01/27 09:22:00 landry Exp $
Using ninja as cmake generator. Avoid calling "make" when "gmake" is expected.
Index: src/librustc_binaryen/build.rs
--- src/librustc_binaryen/build.rs.orig
+++ src/librustc_binaryen/build.rs
@@ -27,6 +27,7 @@ fn main() {
     Config::new("../binaryen")
         .define("BUILD_STATIC_LIB", "ON")
         .build_target("binaryen")
+        .generator("Ninja")
         .build();
 
     // I couldn't figure out how to link just one of these, so link everything.
